client/webpack.config.js

```diff
@@ -115,6 +116,7 @@ const webpackConfig = {
 		devtoolModuleFilenameTemplate: 'app:///[resource-path]',
 	},
 	optimization: {
+		concatenateModules: shouldConcatenateModules,
```

Comment:
won't this also turn on concat for dev builds? I think it's off by default.

Reply:
Excellent catch, @blowery! I didn't realise the defaults were different, but looking at the docs they are. I added an `&&` here, and I think I got the logic right, but please double-check!